search/build.gradle

```diff
@@ -0,0 +1,21 @@
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'kotlin-kapt'
+
+dependencies {
+
+    implementation 'com.android.support.constraint:constraint-layout:1.1.2'
```

Comment:
use common dep in module pleasee, same as support lib. `libs.constraint_layout`

Reply:
Updated.